src/context/mod.rs

```diff
@@ -26,8 +26,8 @@ const LISTENER_PARAM_IDS: Range<u64> = 2..11;
 /// Unique identifier for audio nodes.
 ///
 /// Used for internal bookkeeping.
-#[derive(Debug)]
-pub(crate) struct AudioNodeId(u64);
+#[derive(Debug, Copy, Clone)]
+pub(crate) struct AudioNodeId(pub u64);
```

Comment:
I deliberately marked the u64 as private for type safety. This way users could never construct a malicious NodeId that could crash the render thread. I think if we share the AudioNodeId between control and render thread you do not need to mark it public

Reply:
Yup, see above